test/multi_tenancy.sh

```diff
@@ -32,26 +35,29 @@ delete_users=""
 cleanup_msearch_access=""
 
 function cleanup() {
+    local return_code="$?"
     set +e
-    for user in $cleanup_msearch_access ; do
-        hack_msearch_access $user 2>&1 | artifact_out
-    done
-    for user in $delete_users ; do
-        oc delete user $user 2>&1 | artifact_out
-    done
-    if [ -n "${espod:-}" ] ; then
-        curl_es_pod $espod /project.multi-tenancy-* -XDELETE 2>&1 | artifact_out
-    fi
-    for proj in multi-tenancy-1 multi-tenancy-2 multi-tenancy-3 ; do
-        oc delete project $proj 2>&1 | artifact_out
-        os::cmd::try_until_failure "oc get project $proj" 2>&1 | artifact_out
-    done
-    # this will call declare_test_end, suite_end, etc.
-    os::test::junit::reconcile_output
+	if [ "${DO_CLEANUP:-true}" == "true" ] ; then
+		mkdir -p $ARTIFACT_DIR/$test_name
+		oc -n $LOGGING_NS get configmap fluentd -o jsonpath={.data} --ignore-not-found > $ARTIFACT_DIR/$test_name/fluent-configmap.log ||:
```

Comment:
Nit more then anything but do you need fluent config in context of this test?  Not a blocker for me

Reply:
changed to elasticsearch configmap